whiskey.yml

```diff
@@ -12,6 +12,23 @@ Build:
 - PowerShell:
     OnlyBy: BuildServer
     Path: Save-CarbonSigningKey.ps1
+- PowerShell:
+    ScriptBlock: |
+        $typeDef = @'
+            using System;
+
+            public static class Args
+            {
+                public static void Main(string[] args)
+                {
+                    foreach (var arg in args)
+                    {
+                        Console.WriteLine(arg);
+                    }
+                }
+            }
+        '@

```

Comment:
Why is this duplicating the `Source/Args/Args.cs` file?

Reply:
Oops. Args.cs shouldn't be there.